### PR TITLE
Integrate collision events into GVREventManager

### PR DESCRIPTION
**Pull request by danke-sra**
This allows scripts to handle sensor (collision detector) events.

### CODE REVIEW

**Comment by danke-sra** [comment]
Test patch: https://github.com/gearvrf/GearVRf-Demos/pull/19

File: GVRf/Framework/src/org/gearvrf/GVRBaseSensor.java [hunk 1]
```diff
@@ -20,6 +20,8 @@
 import java.util.List;
 import java.util.Set;
 
+import org.gearvrf.io.GVRInputManager;
+
```
Comment (rahul27) [inline GVRf/Framework/src/org/gearvrf/GVRBaseSensor.java]: Unused import.

**Comment by rahul27** [comment]
@danke-sra ... looks fine to me. I am assuming that you would add IScriptable when you plan to add support for Javascript/Lua.

**Comment by danke-sra** [comment]
@rahul27 Thanks. That will be added later, probably in a demo.

**Pull request merged**